packages/asset-uploader/tsconfig.json

```diff
@@ -1,12 +1,14 @@
-// Extends can't use node's resolution rn https://github.com/dividab/tsconfig-paths/issues/209#issuecomment-1177261091
```

Comment:
this comment is gone but we are still using the workaround for extends with a relative path instead of `extends: "@webstudio-is/tsconfig/base.json"`

Reply:
ye, so at first I've tried to preserve this but it's way easier to treat those files as JSON - where comments are not supported. I've briefly experimented with loading this as JSON5 and manipulating it with [`golden-fleece`](https://github.com/Rich-Harris/golden-fleece) but the results were not satisfactory. Maybe I've just made silly mistake, although its API is pretty straightforward, I've figured out that it's probably more time-efficient to abandon this and just rollback to simple JSONs

Reply:
Oh I didn't realize this. Yeah it seems like its a complication, but that package is supposed to handle it? I can give it a try if you want

Reply:
Ok, so I've tried it again and it seems to work fine. It was producing weird formatting but since I've started to use Prettier here anyway it just fixes it after golden fleece is done with its magic.

You can take a look at my experiment here:
https://github.com/Andarist/webstudio-designer/compare/Andarist:68cd9cc...Andarist:11adbbb

Should I push those commits here?

Reply:
I think its quite important to be able to add comments there over the long run. One option is to use comments as keys if that makes everything easier, not sure this works though here, consumers would have to ignore those and types would need to pass

Reply:
looked at the changes - I see no problem, looks great!

Reply:
just add the comment for autogenerated references and what script generates them

Reply:
Added those comments, the PR is ready for a final review